emails/views.py

```diff
@@ -57,6 +58,7 @@ def _get_user_profile(request, api_token):
 
 
 def _index_POST(request):
+    incr_if_enabled('emails_index_post', 1)
```

Comment:
Should this increment be closer to the `201` response [here](https://github.com/mozilla/fx-private-relay/blob/master/emails/views.py#L88) since this could become a `PUT` or `DELETE` as seen https://github.com/mozilla/fx-private-relay/blob/master/emails/views.py#L69-L72?

Reply:
I moved this under the PUT and DELETE blocks so we don't double-count those as POSTs too.